core/injector/application_injector.cpp

```diff
@@ -490,10 +490,6 @@ namespace {
                                                    std::move(storage),
                                                    std::move(substitutes),
                                                    std::move(block_storage));
-    if (res.has_error()) {
```

Comment:
Are you sure it is safe to remove this check?

Reply:
Yes, `.value()` on next line will throw `exception`.